src/main/graphql/UpdateExportLocation.graphql

```diff
@@ -0,0 +1,4 @@
+mutation updateExportLocation($updateExportLocationInput: UpdateExportLocationInput!) {
```

Comment:
Is this just so we can change the location of the export easily when a final destination is confirmed?

Reply:
It was part of the data meetings the other day I think. We only record this once the export is complete and in S3 so we can see when an export is done.